### PR TITLE
Update Creality Ender-3 V3 SE profile

### PR DESCRIPTION
**Pull request by dim1triy**
- acceleration enabled by default and set acceleration speed
- support enabled by default
- speed_print set to 120
- speed_layer_0 set to 50
- add platform ender3 model

### CODE REVIEW

File: resources/definitions/creality_ender3v3se.def.json [hunk 1]
```diff
@@ -37,19 +38,40 @@
...
+        "machine_max_feedrate_z": { "value": 50 },
         "machine_name": { "default_value": "Creality Ender-3 V3 SE" },
         "machine_start_gcode": { "default_value": "M220 S100 ;Reset Feedrate\nM221 S100 ;Reset Flowrate\n\nG28 ;Home\n\nM420 S1; Enable mesh leveling\n\nG92 E0 ;Reset Extruder\nG1 Z2.0 F3000 ;Move Z Axis up\nG1 X10.1 Y20 Z0.28 F5000.0 ;Move to start position\nM109 S[material_print_temperature_layer_0]\nG1 X10.1 Y145.0 Z0.28 F1500.0 E15 ;Draw the first line\nG1 X10.4 Y145.0 Z0.28 F5000.0 ;Move to side a little\nG1 X10.4 Y20 Z0.28 F1500.0 E30 ;Draw the second line\nG92 E0  ;Reset Extruder\nG1 E-1.0000 F1800 ;Retract a bit\nG1 Z2.0 F3000 ;Move Z Axis up\nG1 E0.0000 F1800 \n" },
         "machine_width": { "default_value": 220 },
         "retraction_amount": { "value": 0.8 },
         "retraction_speed": { "default_value": 40 },
-        "speed_layer_0": { "value": 30 },
-        "speed_print": { "value": 180 }
+        "speed_layer_0": { "value": 50 },
+        "speed_print": { "value": 60 },
```
Comment (zhogov) [inline resources/definitions/creality_ender3v3se.def.json:54]: Why so low? 180 seems to be printing fine for me

Comment (dim1triy) [reply resources/definitions/creality_ender3v3se.def.json:54]: if print PETG, 180mm too much. that is default params, you always can change it in settings

Comment (zhogov) [reply resources/definitions/creality_ender3v3se.def.json:54]: > 180mm too much

What do you mean "too much"?
Ender 3 v3 SE is marketed as able to print 250 mm/s, it should be able to go faster than 60. Sure, we likely want default settings to be conservative, but 60 is too low IMO. Maybe bump it to 100 mm/s?

I did a couple of prints - left is 60 mm/s, right is 180 mm/s. I had to disable minimal layer time and use large test model to make sure speeds are achieved. Prints seem fine:
ender_3v3se_60_vs_180

Problems:

- Horizontal lines on 60 mm/s _seem_ to be caused by the bad flow of the filament as I reuse Ender 3 LACK enclosure without filament feed line
- 180 has slightly worse surface, small horizontal lines that look line under extrusion

Non-default settings: 
Hotend temperature: 230
Initial layer speed: 10 mm/s
Initial layer horizontal expansion: -0.2mm

Comment (dim1triy) [reply resources/definitions/creality_ender3v3se.def.json]: ok. updated speed_print to 100mm/s

**Comment by rotech** [comment]
Just a note, I tried these updates in Cura 5.6.0 with my Ender3 V3SE, and did not have good results from this with the first layer not extruding well (gaps with infill and missing portions of outer wall), and bad dimensional accuracy.  Reverting to the json definitions that come with 5.6.0 and changing nothing else solved all the issues.  Is there other changes in the main branch that are needed to support these definition changes?  Way when I get some time over the next few day I am going to slowly stage these changes in and see if I can pinpoint what setting is causing the issues.

**Comment by dim1triy** [comment]
> Just a note, I tried these updates in Cura 5.6.0 with my Ender3 V3SE, and did not have good results from this with the first layer not extruding well (gaps with infill and missing portions of outer wall), and bad dimensional accuracy. Reverting to the json definitions that come with 5.6.0 and changing nothing else solved all the issues. Is there other changes in the main branch that are needed to support these definition changes? Way when I get some time over the next few day I am going to slowly stage these changes in and see if I can

**Comment by dim1triy** [comment]
> Just a note, I tried these updates in Cura 5.6.0 with my Ender3 V3SE, and did not have good results from this with the first layer not extruding well (gaps with infill and missing portions of outer wall), and bad dimensional accuracy. Reverting to the json definitions that come with 5.6.0 and changing nothing else solved all the issues. Is there other changes in the main branch that are needed to support these definition changes? Way when I get some time over the next few day I am going to slowly stage these changes in and see if I can pinpoint what setting is causing the issues.

Please try to set cool_min_layer_time: 30

**Comment by rotech** [comment]
> > Just a note, I tried these updates in Cura 5.6.0 with my Ender3 V3SE, and did not have good results from this with the first layer not extruding well (gaps with infill and missing portions of outer wall), and bad dimensional accuracy. Reverting to the json definitions that come with 5.6.0 and changing nothing else solved all the issues. Is there other changes in the main branch that are needed to support these definition changes? Way when I get some time over the next few day I am going to slowly stage these changes in and see if I can pinpoint what setting is causing the issues.
> 
> Please try to set cool_min_layer_time: 30

Thanks, I tried that and it did make a small improvement, but still not the same issues.  I also tried removing all the acceleration settings, but that didn't seem to make any difference.  Again just closing Cura, replacing the json file with the one shipped with the 5.6.0 version and back to good quality.  I will keep trying removing settings to see if I can figure it out.

**Comment by dim1triy** [comment]
> > > Just a note, I tried these updates in Cura 5.6.0 with my Ender3 V3SE, and did not have good results from this with the first layer not extruding well (gaps with infill and missing portions of outer wall), and bad dimensional accuracy. Reverting to the json definitions that come with 5.6.0 and changing nothing else solved all the issues. Is there other changes in the main branch that are needed to support these definition changes? Way when I get some time over the next few day I am going to slowly stage these changes in and see if I can pinpoint what setting is causing the issues.
> > 
> > 
> > Please try to set cool_min_layer_time: 30
> 
> Thanks, I tried that and it did make a small improvement, but still not the same issues. I also tried removing all the acceleration settings, but that didn't seem to make any difference. Again just closing Cura, replacing the json file with the one shipped with the 5.6.0 version and back to good quality. I will keep trying removing settings to see if I can figure it out.

Please try speed_layer_0 to old value 30, these two parameters influence first layer

**Comment by rotech** [comment]
> > > > Just a note, I tried these updates in Cura 5.6.0 with my Ender3 V3SE, and did not have good results from this with the first layer not extruding well (gaps with infill and missing portions of outer wall), and bad dimensional accuracy. Reverting to the json definitions that come with 5.6.0 and changing nothing else solved all the issues. Is there other changes in the main branch that are needed to support these definition changes? Way when I get some time over the next few day I am going to slowly stage these changes in and see if I can pinpoint what setting is causing the issues.
> > > 
> > > 
> > > Please try to set cool_min_layer_time: 30
> > 
> > 
> > Thanks, I tried that and it did make a small improvement, but still not the same issues. I also tried removing all the acceleration settings, but that didn't seem to make any difference. Again just closing Cura, replacing the json file with the one shipped with the 5.6.0 version and back to good quality. I will keep trying removing settings to see if I can figure it out.
> 
> Please try speed_layer_0 to old value 30, these two parameters influence first layer

Thank you, but this didn't help either.  I've also had a look back in the history of this file and there are a few other updates that may have caused/contributed to the problem that I will look at.  I have eliminated a few of the changes , I am adding them into my good config one at a time.  This is quite time consuming though.  I don't know if my issue should necessarily block this PR, as I don't know if anyone else is having the same problem.

**Comment by hugglesfox** [comment]
Just thought to throw my 2 cents. I was having issues with bed adhesion on tight corners (seemed like the print head was accelerating too fast) and using these settings helped fix that.

**Comment by Izzy5656** [comment]
The default ender3 v3 se profile has 'group outer walls' under experimental checked by default. This caused under extrusion when doing a long travel to another outer wall. Hopefully this is the right place to surgest this issue

**Comment by GregValiant** [comment]
Changing a definition file per personal preferences means everyone gets stuck with them.  For instance, I think my personal StartUp Gcode is terrific, but I'm sure not everyone would find it to be so.

Better to copy your personal definition file to the "Definitions" folder in the Config Folder and use custom overrides to set Cura up the way you want.
This is one of my overrides.
	"group_outer_walls": { "value": false }

**Pull request closed without merging**